src/datasets/providers/statsig.ts

```diff
@@ -0,0 +1,15 @@
+import StatsigNoFillSvg from '@site/static/img/statsig-no-fill.svg';
+import { Provider } from '.';
+
+export const Statsig: Provider = {
+  name: 'Statsig',
+  logo: StatsigNoFillSvg,
+  technologies: [
+    {
+      technology: '.NET',
+      vendorOfficial: false,
```

Comment:
Please let me know if this should be marked as vendor official.

Reply:
I have reached out to Statsig and informed them of this work, but this is not an official provider.

Reply:
What benefit is there to being considered "vendorofficial"?  Or what other implications does that come with?  Does that require us to maintain the package?

Reply:
It's a way for end users to know if the vendors themselves are supporting the providers by addressing bug fixes and maintenance efforts. In the near future, an official providers will be a requirement for listing a vendor on a commercial offering page.